src/pages/components/text.mdx

```diff
@@ -12,39 +12,54 @@ thumbnail: '/assets/images/components/thumbnail-text.svg'
 
 ## Usage
 
-- テキストのスタイルに困った場合、シンプルに `<Text>` でテキストを囲ってください
-  - `props` を指定しなくとも、本文で使用するスタイル（`type='body'`、`size='md'`）がデフォルトで適用されています
-- デフォルトでは、`<p>` 要素としてレンダリングされます
+- テキストのスタイルに困った場合、シンプルに`<Text>`でテキストを囲ってください
```

Comment:
I cut off the space before and after the code (since margin is attached even in css).